Opta-Stats/OptaStats.md

```diff
@@ -0,0 +1,82 @@
+# Opta Stats
+
+*Built by LATAM Team*
+
+**Supports:** *iOS and Android*
+*Current Version: 0.2.22 (Android), 1.0.52 (iOS)*
+
+## About
+The Opta Statistic Plugin is being developed primarily to show the statistics for Copa America 2019 tournament. Since Opta is the main provider for several events in LATAM we are pivoting this plugin to be general.
+
+## URL Schemes supported
+We defined a set of pre-configured number of screens which you can open through a URL scheme from anywhere in the application. In order to do so, we need to define a well-known URL schema. These are the proposed schemas for this integration.
```

Comment:
replace "We defined a set of pre-configured number of screens" with "We predefined a set of screens"